packages/itwin/tree-widget/src/components/trees/VisibilityTreeRenderer.tsx

```diff
@@ -50,12 +87,12 @@ export const createVisibilityTreeNodeRenderer = (iconsEnabled: boolean, descript
  * Checkbox renderer that renders an eye.
  * @public
  */
-export const visibilityTreeNodeCheckboxRenderer = (props: NodeCheckboxRenderProps) => (
+export const VisibilityTreeNodeCheckboxRenderer = (props: NodeCheckboxRenderProps) => (
```

Comment:
Maybe rename this to `VisibilityTreeNodeCheckbox`?

Reply:
Fixed